main.tf

```diff
@@ -60,7 +60,29 @@ resource "spacelift_policy" "custom" {
   name = format("%s %s Policy", upper(split(".", each.key)[0]), title(replace(split(".", each.key)[1], "-", " ")))
```

Comment:
Have you tested this out in infra-live to make sure it works as expected?

Reply:
this did take some tests. I think it's behaving cleanly now.... at least locally. About to try on cplive